src/commands/config/prompts.js

```diff
@@ -5,7 +5,8 @@ export const CONFIGURATION_PROMPT_NAMES = {
   AUTO_ADD: 'autoAdd',
   EMOJI_FORMAT: 'emojiFormat',
   SCOPE_PROMPT: 'scopePrompt',
-  SIGNED_COMMIT: 'signedCommit'
+  SIGNED_COMMIT: 'signedCommit',
+  SKIP_PROMPTING_FILLED_INFO: 'skipPromptingFilledInfo'
```

Comment:
Same as for this option, is a little bit ambiguous

Reply:
Right, do you have some suggestion ?